corehq/apps/app_manager/views.py

```diff
@@ -1536,8 +1538,12 @@ def edit_module_detail_screens(req, domain, app_id, module_id):
         # Note that we use the empty tuple as the sentinel because a filter
         # value of None represents clearing the filter.
         detail.short.filter = filter
-    if custom_xml != None:
+    if custom_xml is not None:
         detail.short.custom_xml = custom_xml
+    if use_case_tiles is not None and short is not None:
+        detail.short.use_case_tiles = use_case_tiles
+    if persist_tile_on_forms is not None and short is not None:
+        detail.short.persist_tile_on_forms = persist_tile_on_forms
```

Comment:
Looks like these two if blocks could be moved up under line 1530 `if short is not None:`, and then they would not need to repetitively check that condition.